BossMod/Autorotation/AutoHints.cs

```diff
@@ -7,6 +7,8 @@ namespace BossMod
     // this is used e.g. in outdoor or on trash, where we have no active bossmodules
     public class AutoHints : IDisposable
     {
+        static readonly float RAIDWIDE_SIZE = 40;
```

Comment:
VBM uses standard C# naming conventions: https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/identifier-names

Specifically:
* Use PascalCase for constant names, both fields and local constants.